tests/align/talign.nim

```diff
@@ -39,15 +43,13 @@ proc foobar() =
   doAssert (cast[uint](addr(mylocal2)) and 127) == 0
   doAssert (cast[uint](addr(mylocal3)) and 127) == 0
 
-  echo "align ok"
-
 foobar()
 
-# bug #13122
+block issue_13122:
```

Comment:
Put this second test inside a block. 
Sometimes i'm worried this changes the semantics, since top-level seems to be treated differently, but it doesn't look like the case here.

Reply:
A `block:` (`static:` too) only opens a new scope, the content is still treated as top-level.

Top-level variables (globals) have various differences in semantics compared to local variables. For example, they're not captured in closures and are copied into `sink` arguments (instead of moved) by default.

Reply:
TIL